app/services/base_service.rb

```diff
@@ -0,0 +1,9 @@
+class BaseService
+  def self.json(response)
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  # def self.conn(url)
```

Comment:
Use of this method in the child class MapService raised the following error:

```
  1) Map Service retrieves latitude and longitude for a city
     Failure/Error: JSON.parse(response.body, symbolize_names: true)

     JSON::ParserError:
       784: unexpected token at 'The AppKey submitted with this request is invalid.'
     # ./app/services/base_service.rb:3:in `json'
     # ./app/services/map_service.rb:7:in `get_latitude_and_longitude'
     # ./spec/services/map_service_spec.rb:11:in `block (2 levels) in <top (required)>'

You've Nyaned for 0.6 seconds


Finished in 0.60125 seconds (files took 3.27 seconds to load)
5 examples, 1 failure, 1 pending
```

I wasn't sure how to resolve this so I commented this out for now. I'll return to this if I have time after more functionality is fleshed out.